ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/EthSchedulerShutdownTest.java

```diff
@@ -55,9 +55,7 @@ public void shutdown_syncWorkerShutsDown() throws InterruptedException {
     ethScheduler.stop();
 
     assertThat(syncWorkerExecutor.isShutdown()).isTrue();
-    assertThat(syncWorkerExecutor.isTerminated()).isFalse();
```

Comment:
Do we want to test for multiple tasks in the queue and assert they don't get executed?

Reply:
Good idea.  Added.